src/Data/DataResponse.php

```diff
@@ -12,7 +14,7 @@
  * For example, `['name' => 'Dmitriy']` to be formatted to JSON using {@see \Yiisoft\Yii\Web\Data\Formatter\JsonDataResponseFormatter}
  * when {@see DataResponse::getBody()} is called.
  */
-class DataResponse implements ResponseInterface
+final class DataResponse implements ResponseInterface
```

Comment:
I think that here @yiiliveext left the opportunity to extend the decorator

Reply:
> I think that here @yiiliveext left the opportunity to extend this decorator

We will remove "final" when necessary.